stdlib/LinearAlgebra/src/blas.jl

```diff
@@ -797,6 +806,23 @@ for (fname, elty) in ((:zhemv_,:ComplexF64),
     end
 end
 
+"""
+    hemv(ul, alpha, A, x)
+
+Return `alpha*A*x`. `A` is assumed to be Hermitian.
+Only the [`ul`](@ref stdlib-blas-uplo) triangle of `A` is used.
+`alpha` is a scalar.
+"""
+hemv(ul, alpha, A, x)
```

Comment:
Is this not trying to call the function?

Reply:
I don't think so. I took this from other places, where it seemed to work fine, and serves to distinguish the different signatures. But I noticed that there is no common style in the file.